plugins/BEdita/Core/src/Search/Adapter/SimpleAdapter.php

```diff
@@ -0,0 +1,150 @@
+<?php
+declare(strict_types=1);
+
+/**
+ * BEdita, API-first content management framework
+ * Copyright 2023 ChannelWeb Srl, Chialab Srl
+ *
+ * This file is part of BEdita: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * See LICENSE.LGPL or <http://gnu.org/licenses/lgpl-3.0.html> for more details.
+ */
+namespace BEdita\Core\Search\Adapter;
+
+use BEdita\Core\Exception\BadFilterException;
+use BEdita\Core\Search\BaseAdapter;
+use Cake\Database\Expression\FunctionExpression;
+use Cake\Database\Expression\QueryExpression;
+use Cake\Datasource\EntityInterface;
+use Cake\ORM\Query;
+use Cake\Utility\Hash;
+use Cake\Validation\Validator;
+
+/**
+ * Simple search adapter.
+ * Execute search via `LIKE` query.
+ *
+ * @since 5.14.0
+ */
+class SimpleAdapter extends BaseAdapter
+{
+    /**
+     * Get validator.
+     *
+     * @param array $config Search configuration
+     * @return \Cake\Validation\Validator
+     */
+    public function getValidator(array $config = []): Validator
```

Comment:
Since this method is not part of the public interface, I'd make it protected instead.

```suggestion
    protected function getValidator(array $config = []): Validator
```